src/hash_from_two_arrays.rs

```diff
@@ -0,0 +1,13 @@
+// http://rosettacode.org/wiki/Hash_from_two_arrays
+
+use std::collections::HashMap;
+
+fn main() {
+    let keys = ["a", "b", "c"];
+    let values = [1, 2, 3];
+
+    let mut hash = HashMap::new();
+    for (key, value) in keys.iter().zip(values.iter()) {
+        hash.insert(key, value);
+    }
```

Comment:
Note that you could also use `collect`, as shown below:

`let hash: HashMap<_, _> = keys.iter().zip(values.iter()).collect();`

Which version do you prefer?